src/rai_extensions/rai_nomad/rai_nomad/nomad.py

```diff
@@ -0,0 +1,348 @@
+# Copyright (C) 2024 Robotec.AI
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from typing import Tuple
+
+import numpy as np
+import rclpy
+import torch
+import yaml
+from ament_index_python.packages import get_package_share_directory
+from diffusers.schedulers.scheduling_ddpm import DDPMScheduler
+from geometry_msgs.msg import Twist
+from rcl_interfaces.msg import ParameterDescriptor, ParameterType
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.node import Node
+
+# ROS
```

Comment:
Since we are using isort, these comments may at some point become confusing. In fact, at this point they are confusing, as ROS 2 things are imported both preceding and following the comment.

Reply:
I removed those comments.